src/ontology/mondo-edit.obo

```diff
@@ -244551,12 +244550,13 @@ subset: otar {source="MONDO:OTAR"}
 subset: rare
 synonym: "ADEAF" EXACT ABBREVIATION [Orphanet:101046]
 synonym: "ADLTE" EXACT ABBREVIATION [Orphanet:101046]
+synonym: "adolescent/adult onset autosomal dominant epilepsy with auditory features" RELATED [https://www.epilepsydiagnosis.org/syndrome/adeaf-overview.html]
 synonym: "ADPEAF" EXACT ABBREVIATION [Orphanet:101046]
 synonym: "Autosomal dominant epilepsy with auditory features" EXACT CLINGEN_LABEL [icd11.foundation:832717248, Orphanet:101046]
 synonym: "autosomal dominant epilepsy with auditory features" EXACT [icd11.foundation:832717248, Orphanet:101046]
 synonym: "autosomal dominant lateral temporal lobe epilepsy" EXACT [Orphanet:101046]
 synonym: "autosomal dominant partial epilepsy with auditory features" RELATED [GARD:0002257]
-synonym: "epilepsy, familial temporal lobe 1" NARROW []
+synonym: "autosomal dominant partial/lateral temporal epilepsy with auditory features" EXACT [https://www.epilepsydiagnosis.org/syndrome/adeaf-overview.html]
```

Comment:
is the synonym scope here correct?

Reply:
From ILAE, it says "EAF may occur as a familial syndrome, familial EAF (FEAF, previous known as autosomal dominant lateral temporal lobe epilepsy or **autosomal dominant partial epilepsy with auditory features**)", so this should probably be EXACT DEPRECATED too, do you agree?

Reply:
yes,I agree: exact - historical would also be how I would curate it

Reply:
updated 🦖